src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -2599,6 +2599,14 @@ public String save() {
                 if (nNewFiles > 0) {
                     // Save the NEW files permanently and add the to the dataset: 
```

Comment:
It would be nice to change this from "the" to "them" while we're in here but whatever.